### DISCUSSION TITLE
problem with AVM fritzbox wiregurad config

### DISCUSSION THREAD

**Discussion by er34zer** [RaspAP/raspap-webgui#1578] [Q&A]
Hi,

every time load the config file from a Fritzbox, i get this massege.

![grafik](https://github.com/RaspAP/raspap-webgui/assets/169102537/cfbc6afe-7cb1-45e7-ae2b-e50636a3fde0)

after that no new .conf can be loaded. the only way is > format SD > load RaspAP via SSH...

is there some one with the same problem?

**Comment by billz** [comment]
Please submit a bug report. Be sure to provide as much detail as possible in the "Steps to reproduce" section, including the content of the stack trace. Thanks
